tarea/class-8-task.js

```diff
@@ -0,0 +1,67 @@
+const $form = document.querySelector('[name=formulario]');
+$form.onsubmit = validateForm;
+
+function validateForm(e) {
+    e.preventDefault();
+    const city = $form['ciudad'].value;
+    const giftDescription = $form['descripcion-regalo'].value;
+    const errorCity = validateCity(city);
+    const errorGiftDescription = validateGiftDescription(giftDescription);
+
+    const errors = {
+        ciudad: errorCity,
+        'descripcion-regalo': errorGiftDescription
+    }
+
+    const isSuccess = handleErrors(errors) === 0;
+    if (isSuccess) {
+        $form.className = 'oculto';
+        document.querySelector('#exito').className = '';
+
+        setTimeout(() => {
+            window.location.href = 'wishlist.html';
+        }, 5000);
```

Comment:
Save this number in a constant like ``DELAY_IN_MILLISECONDS``

Reply:
Where should I add the constant?
Inside the setTimeout on line 22? 
Or outside on line 20?

Reply:
Like this:
```
const DELAY_IN_MILLISECONDS = 5000;
        setTimeout(() => {
            window.location.href = 'wishlist.html';
        }, DELAY_IN_MILLISECONDS);```

Reply:
I appreciate it!
 i will change it now